doc/howto/network_bgp.md

```diff
@@ -68,6 +68,7 @@ Set the following configuration options on the uplink network:
 - `bgp.peers.<name>.address` - the peer address to be used by the downstream networks
 - `bgp.peers.<name>.asn` - the {abbr}`ASN (Autonomous System Number)` for the local server
 - `bgp.peers.<name>.password` - an optional password for the peer session
+- `bgp.peers.<name>.holdtime` - an optional hold time for the peer session (in seconds; default is 3 minutes)
```

Comment:
This is a bit confusing in my opinion. Elsewhere we say the default is 180s and here we say the default is 3 minutes.
I realize they are the same, but it still feels a bit odd to be mixing units like that.
Also if we ever change the default then it'll likely be missed.
So perhaps just remove the default statement here and leave that to the reference table itself.

Reply:
Agreed, this notification is simplified with all comments too.